src/main/app/src/app/zaken/zaak-view/zaak-view.component.html

```diff
@@ -181,12 +180,12 @@
                                                           [verlengReden]="zaak.redenVerlenging"
                                                           [verlengDuur]="zaak.duurVerlenging"
                                                           [verlengingstermijn]="zaak.zaaktype.verlengingstermijn"
-                                                          [readonly]="!zaak.rechten.wijzigen"
+                                                          [readonly]="!zaak.rechten.wijzigen || zaak.isProcesGestuurd"
                                                           [actieOpschorten]="zaak.zaaktype.opschortingMogelijk && zaak.isOpen && !zaak.isHeropend
-                                                                         && !zaak.isOpgeschort && zaak.rechten.behandelen"
-                                                          [actieHervatten]="zaak.isOpgeschort && zaak.rechten.behandelen"
+                                                                         && !zaak.isOpgeschort && zaak.rechten.behandelen && !this.zaak.isProcesGestuurd"
+                                                          [actieHervatten]="zaak.isOpgeschort && zaak.rechten.behandelen && !this.zaak.isProcesGestuurd"
```

Comment:
Moet die misschien in OPA geregeld worden?

Reply:
Dit is geen recht maar een eigenschap van de zaak